pdb.py

```diff
@@ -1457,6 +1457,29 @@ def do_down(self, arg='1'):
     do_down.__doc__ = pdb.Pdb.do_down.__doc__
     do_d = do_down
 
+    def do_top(self, arg):
+        if self.curindex == 0:
+            print('*** Oldest frame', file=self.stdout)
```

Comment:
Could use `self.error()`, and return (for less indent below).
Same below.

Reply:
Changed in https://github.com/pdbpp/pdbpp/pull/307/commits/7f3073c9f520e672a8bc331ddf52a5a30d084ebd